server/sandbox_run_linux.go

```diff
@@ -365,9 +365,10 @@ func (s *Server) runPodSandbox(ctx context.Context, req *types.RunPodSandboxRequ
 	}
 
 	resourceCleaner := resourcestore.NewResourceCleaner()
+	noNeedToCleanup := false
```

Comment:
can this be `storeResource` or something, and can there be a comment explaining why we're doing this?

Reply:
Thank you for your suggestion. I have resubmitted it